modules/api/api-xsjs/src/main/resources/META-INF/dirigible/xsk/web/web.js

```diff
@@ -375,11 +380,11 @@ exports.WebResponse = function (clientResponse) {
       syncCookies.bind(this)();
 
       if (content instanceof Array && this.contentType === 'application/zip') {
-        var parsedContent = JSON.parse(bytes.byteArrayToText(content));
-        var outputStream = dResponse.getOutputStream();
+        const parsedContent = JSON.parse(bytes.byteArrayToText(content));
+        const outputStream = dResponse.getOutputStream();
         if (outputStream.isValid()) {
           try {
-            var zipOutputStream = zip.createZipOutputStream(outputStream);
+            let zipOutputStream = zip.createZipOutputStream(outputStream);
```

Comment:
Use `const`

Reply:
Methods createZipEntry() and writeText() are later executed on zipOutputStream.

Reply:
So what if they're executed? `const` means you can't reassign the variable